pci_dss_v40/appendix_a3.pp

```diff
@@ -0,0 +1,160 @@
+locals {
+  pci_dss_v40_appendix_a3_common_tags = merge(local.pci_dss_v40_common_tags, {
+    control_set = "3"
```

Comment:
@Priyanka-Chatterjee-2000 could you please verify if the `control_set = <value>` is required or not?